v0x04/match_fields.py

```diff
@@ -54,10 +57,14 @@ def as_of_tlv(self):
     @classmethod
     def from_of_tlv(cls, tlv):
         """Return an instance from a pyof OXM TLV."""
-        vlan_id = int.from_bytes(tlv.oxm_value[:2], 'big') & 4095
+        vlan_id = int.from_bytes(tlv.oxm_value[:2], 'big')
```

Comment:
Aldo, I'll encourage you and your future self to add some extra docstrings especially for this case to easily remember in a very summarized way the cases where supporting and why we're checking if vlan_id is diff than 4096.

Reply:
Docstrings added commit [7b4a04f45c2b39d51d693ac0729f6c88ad4c5e02](https://github.com/kytos-ng/of_core/pull/98/commits/7b4a04f45c2b39d51d693ac0729f6c88ad4c5e02)

Reply:
Looks good. It's nice to have this.